src/main/java/zd/zero/waifu/motivator/plugin/WaifuMotivatorProject.java

```diff
@@ -76,22 +76,22 @@ private void updatePlatformStartupConfig() {
     private void initializeStartupMotivator() {
         if ( isMultipleProjectsOpened() ) return;
 
-        AlertConfiguration config = AlertConfiguration.builder()
-            .isAlertEnabled( pluginState.isStartupMotivationEnabled() || pluginState.isStartupMotivationSoundEnabled() )
-            .isDisplayNotificationEnabled( pluginState.isStartupMotivationEnabled() )
-            .isSoundAlertEnabled( pluginState.isStartupMotivationSoundEnabled() )
-            .build();
-        WaifuMotivatorAlert motivatorAlert = WaifuMotivatorAlertFactory
-            .createAlert(
-                project,
-                AlertAssetProvider.getRandomAssetByCategory( NEUTRAL ),
-                config
-            );
+        AlertConfiguration config = new AlertConfiguration(
```

Comment:
You'll see this a bunch, kotlin and lombok do not play nice. So I converted anything that Kotlin code needed to use into kotlin.

IMO: Kotlin > Java + Lombok

Reply:
I'm now starting to like Kotlin, I'll make time to learn it so please bear with me for being incompetent 😅

Reply:
Based on how you've been using Java here, I'm pretty sure you'll like Kotlin :)